Dockerfile

```diff
@@ -1,22 +1,18 @@
 # Mozilla Kinto server
-
-FROM node:lts-bullseye-slim as node-builder
-RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates curl
-COPY scripts/build-kinto-admin.sh .
-COPY /kinto/plugins/admin ./kinto/plugins/admin
-RUN bash build-kinto-admin.sh
-
 FROM python:3.10-slim-bullseye as python-builder
 RUN python -m venv /opt/venv
-RUN apt-get update && apt-get install -y --no-install-recommends build-essential libpq-dev
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential libpq-dev ca-certificates curl
 ARG KINTO_VERSION=1
 ENV SETUPTOOLS_SCM_PRETEND_VERSION_FOR_KINTO=${KINTO_VERSION} \
     PATH="/opt/venv/bin:$PATH"
 # At this stage we only fetch and build all dependencies.
 WORKDIR /pkg-kinto
-COPY constraints.txt .
-COPY pyproject.toml .
+COPY constraints.txt pyproject.toml ./
```

Comment:
We could `pip install --upgrade pip && pip install -r constraints.txt` after this layer, again for cache preservation purposes. 

Similar to my other comment, right now, any time we change a Kinto source file, we'll need to upgrade pip, then reinstall all of the dependencies. If we move this step up here, we'll only need to do that when `constraints.txt` or `pyproject.toml` change.

Reply:
added